manim/utils/color.py

```diff
@@ -55,6 +55,9 @@
     "RED",
     "RED_B",
     "RED_A",
+    "MANIMCE_LOGO_BLUE",
+    "MANIMCE_LOGO_GREEN",
+    "MANIMCE_LOGO_RED",
```

Comment:
I would prefer to not have them here in the colourspace.
Maybe It would be nice to document them somewhere (e.g. in the docs, like in the example I pointed out here:
https://github.com/ManimCommunity/manim/pull/649/files#r518272186), however, I do not think that it is worth introducing only these 3 colours as MANIMCE_LOGO_COLOURs. Probably when we have something like a completely new colourmap they can be included.

Reply:
Out of curiosity: why not have them here?

Reply:
Are they really that important, that they will be imported into the main manim namespace?
I think they will be barely used, and it probably makes more sense to include them as soon as we have other colormap options

Reply:
I could say the same thing about a lot of the other colors that are already there :sweat_smile: Also, you can be sure that they will never be used *unless they are added in here*.

Ultimately, I'm neutral about this. No harm either way I think

Reply:
I wanted to use this as a nice namesake, but in second thought it could be more useful to define the color and show that it's a string of RGB 8 bit color/hex instead. Probably makes for a more meaningful example. I do like the idea of a giant colormap/color palette plugin though

Reply:
> I could say the same thing about a lot of the other colors that are already there
Good point!
I would also find it good to only have the standard colors GREEN, BLUE, YELLOW, etc, and not all of their nuances.
But my opinion on that is also not too strong. I am also fine with adding MANIMCE_LOGO_BLUE etc.